sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -1,35 +1,92 @@
+    icon="fa:fas fa-check-circle"
+    icon-color="success"
+    :background-color="hasDisbursementEvent ? undefined : 'success-bg'"
+    content="StudentAid BC has determined an outcome with 1 or more of your requested change. Please review your new assessment in the table below."
+    v-if="
+      assessmentDetails.assessmentTriggerType ===
+      AssessmentTriggerType.StudentAppeal
+    "
+  />
+  <!-- Scholastic standing changed -->
+  <application-status-tracker-banner
+    label="You have a new assessment due to your scholastic standing"
+    icon="fa:fas fa-check-circle"
+    icon-color="success"
+    :background-color="hasDisbursementEvent ? undefined : 'success-bg'"
+    content="Your institution informed us of your scholastic standing, which changed your assessment evaluation. Please review your new assessment in the table below. You can also click “View request” to see the details from your institution. Please contact the Financial Aid Officer from your institution, if you have questions about your scholastic standing."
+    v-if="
+      assessmentDetails.assessmentTriggerType ===
+      AssessmentTriggerType.ScholasticStandingChange
+    "
+  />
+  <!-- Offering changed -->
+  <application-status-tracker-banner
+    label="You have a new assessment due to an update with your study period"
+    icon="fa:fas fa-check-circle"
+    icon-color="success"
+    :background-color="hasDisbursementEvent ? undefined : 'success-bg'"
+    content="Your institution updated the study period that was submitted with your application, which changed your assessment evaluation. Please review your new assessment in the table below. If you have concerns or require more information, please contact the Financial Aid Officer from your institution."
+    v-if="
+      assessmentDetails?.assessmentTriggerType ===
+      AssessmentTriggerType.OfferingChange
+    "
+  />
+  <!-- Disbursement/COE banners -->
   <multiple-disbursement-banner
-    v-if="applicationCOEDetails?.secondCOE"
-    :firstCOEStatus="applicationCOEDetails?.firstCOE?.coeStatus"
-    :secondCOEStatus="applicationCOEDetails?.secondCOE?.coeStatus"
+    v-if="assessmentDetails?.secondDisbursement"
+    :firstCOEStatus="assessmentDetails?.firstDisbursement?.coeStatus"
+    :secondCOEStatus="assessmentDetails?.secondDisbursement?.coeStatus"
     :coeDenialReason="multipleCOEDenialReason"
     :firstDisbursementStatus="
-      applicationCOEDetails?.firstCOE?.disbursementScheduleStatus
+      assessmentDetails?.firstDisbursement?.disbursementScheduleStatus
     "
     :secondDisbursementStatus="
-      applicationCOEDetails?.secondCOE?.disbursementScheduleStatus
+      assessmentDetails?.secondDisbursement?.disbursementScheduleStatus
     "
   />
   <disbursement-banner
     v-else
-    :coeStatus="applicationCOEDetails?.firstCOE?.coeStatus"
-    :coeDenialReason="applicationCOEDetails?.firstCOE?.coeDenialReason"
+    :coeStatus="assessmentDetails?.firstDisbursement?.coeStatus"
```

Comment:
for my knowlede, what is `multiple-disbursement-banner` and `disbursement-banner`

Reply:
They represent the cards related to the Disbursement Status and COE Status. They are shared between enrollment status and completed status.
![image](https://user-images.githubusercontent.com/61259237/224450679-6a0074e9-f304-408d-adfa-51116d28e3b0.png)